### ISSUE TITLE
Process files with not ".php" extension

### ISSUE THREAD

**Issue by mxr576** [rectorphp/rector#1188]
As I can see in the ProcessCommand's implementation it only processes files with `*.php` extension from the provided source directories. It would be great if this could be configured with a parameter.

```sh
$ ls foo/
bar.inc     bar.module  bar.php
```
Actual:

```sh
$ ./vendor/bin/rector process --dry-run foo -v
Rector v0.4.4

/var/www/html/foo/bar.php
/var/www/html/foo/bar.php
/var/www/html/foo/bar.php                                                                                                                       
 [OK] Rector is done! 0 changed files                                                                                   
```

Expected:

```sh
$ ./vendor/bin/rector process --dry-run foo -v
Rector v0.4.4

/var/www/html/foo/bar.inc
/var/www/html/foo/bar.module
/var/www/html/foo/bar.php
/var/www/html/foo/bar.inc
/var/www/html/foo/bar.module
/var/www/html/foo/bar.php
/var/www/html/foo/bar.inc
/var/www/html/foo/bar.module
/var/www/html/foo/bar.php
                                                                                                                       
 [OK] Rector is done! 0 changed files                                                                                   
```

**Comment by TomasVotruba** [comment]
Hi, I just noticed this issue.
Do you have some idea to solve it?

**Comment by mxr576** [comment]
@TomasVotruba I also like that approach. What do you think about my code in #1285? Is it too simple? =]

**Comment by TomasVotruba** [comment]
> Is it too simple? =]

Well, the simpler the better :+1:

**Comment by TomasVotruba** [comment]
Resolved by #1285
